web-common/src/features/entity-management/RenameAssetModal.svelte

```diff
@@ -20,36 +18,46 @@
     VALID_NAME_PATTERN,
     isDuplicateName,
   } from "./name-utils";
+  import { superForm, defaults } from "sveltekit-superforms";
+  import { yup } from "sveltekit-superforms/adapters";
+  import { object, string } from "yup";
 
   export let closeModal: () => void;
   export let filePath: string;
   export let isDir: boolean;
 
   let error: string;
-  const queryClient = useQueryClient();
-
-  $: runtimeInstanceId = $runtime.instanceId;
-  $: allNamesQuery = useAllFileNames(queryClient, runtimeInstanceId);
 
   const [folder, assetName] = splitFolderAndName(filePath);
 
-  $: existingDirectories = useDirectoryNamesInDirectory(
-    runtimeInstanceId,
-    folder,
-  );
+  const validationSchema = object({
+    newName: string()
+      .required("Enter a name!")
+      .matches(VALID_NAME_PATTERN, INVALID_NAME_MESSAGE),
+  });
```

Comment:
I can't seem to trigger this validation message. When I leave the form empty, then try to submit it, I don't see the message "Enter a name!" pop up anywhere.

Reply:
Unifying the Input components meant bringing over the conditionally shown error behavior from InputV2, I believe. One of the conditionals is that there exists a value in the input. Obviously that's in conflict with an error for a missing value... Turned on the alwaysShowError prop for now, though I may consider removing this conditional behavior as I convert more forms.